Frida/Script.swift

```diff
@@ -204,6 +204,43 @@ public class Script: NSObject, NSCopying {
             }
         }
 
+        if let dict = message as? [String: Any],
```

Comment:
This is guaranteed by the Frida API, so we should not be handling messages of any other shape.